src/pages/Wallet/SendPage.tsx

```diff
@@ -126,55 +206,99 @@ const SendPage = () => {
 
   return (
     <>
-      <LogoContent>
-        <SendLogo>
-          {isSending ? <Spinner size="30%" /> : <Send color={theme.global.accent} size={'70%'} strokeWidth={0.7} />}
-        </SendLogo>
-      </LogoContent>
+      {isSending ? (
+        <HeaderLogo>
+          <Spinner size="30%" />
+        </HeaderLogo>
+      ) : (
+        !isChecking && (
+          <HeaderContent>
+            <HeaderLogo>
+              <Send color={theme.global.accent} size={'70%'} strokeWidth={0.7} />
+            </HeaderLogo>
+          </HeaderContent>
+        )
+      )}
+
       {!isChecking ? (
-        <Section>
-          <Input
-            placeholder="Recipient's address"
-            value={address}
-            onChange={(e) => handleAddressChange(e.target.value)}
-            error={addressError}
-            isValid={address.length > 0 && !addressError}
-          />
-          <Input
-            placeholder="Amount"
-            value={amount}
-            onChange={(e) => handleAmountChange(e.target.value)}
-            type="number"
-            min="0"
-          />
-        </Section>
+        <>
+          <Section>
+            <Input
+              placeholder="Recipient's address"
+              value={address}
+              onChange={handleAddressChange}
+              error={addressError}
+              isValid={address.length > 0 && !addressError}
+            />
+            <Input placeholder="Amount (ℵ)" value={amount} onChange={handleAmountChange} type="number" min="0" />
+            {expectedFeeInALPH && <InfoBox short label="Expected fee" text={`${expectedFeeInALPH}ℵ`} />}
+          </Section>
+          <ExpandableSection sectionTitle="Advanced settings">
+            <Input
+              id="gas-amount"
+              placeholder="Gas amount"
+              value={gasAmount}
+              onChange={handleGasAmountChange}
+              type="number"
+              error={gasAmountError}
+            />
```

Comment:
Shouldn't the `gas-amount` number input also have a `min` attribute, like the `gas-price` below?